### ISSUE TITLE
Examples WebGPU: Some examples fail due to missing assets.

### ISSUE THREAD

**Issue by marlyx** [bevyengine/bevy-website#641]
Some examples on the "Bevy Examples in WebGPU" page fail due to failed asset loading. (404 on fetch). 

Relevant system information:
 - Browser: Chrome 113.0.5672.126 (Official Build) (arm64)
 - Computer: MacBookPro18,1 (Apple M1 Pro), macOS 13.4 (22F66)

I have checked all examples on the page and currently these fail due to 404 errors:

https://bevyengine.org/examples-webgpu/2D%20Rendering/custom-gltf-vertex-attribute/
    Fetchcustom_gltf_2d.wgsl
    Fetchbarycentric.gltf

https://bevyengine.org/examples-webgpu/3D%20Rendering/parallax-mapping/
    Fetchcube_normal.png
    Fetchcube_color.png
    Fetchcube_depth.png

https://bevyengine.org/examples-webgpu/Shaders/post-processing/
    post_processing.wgsl

https://bevyengine.org/examples-webgpu/Tools/scene-viewer/
    FlightHelmet.gltf

**Comment by mockersf** [comment]
yup, you're right. The website is mostly about Bevy `latest` branch, except for the WebGPU example page that is built from the `main` branch for now, but will be built from the `latest` branch after next release... I guess we could copy assets from the main branch for now even for WebGL2 examples, they should still work

**Comment by mockersf** [comment]
should be fixed now. Two of them still fail but for other reasons: custom-gltf-vertex-attribute and scene-viewer

**Comment by lomirus** [comment]
There are still some example assets responded 404: <https://bevyengine.org/examples/3d-rendering/volumetric-fog/>

![Image](https://github.com/user-attachments/assets/b8175fb3-9b10-4f04-9c30-50c512dc7e7a)

**Comment by mockersf** [comment]
it's on the .meta files which are expected to 404
